### ISSUE TITLE
Virus detected!

### ISSUE THREAD

**Issue by sylvestersl** [YTEC-info/CH341A-Softwares#13]
Please, remove all Virus detected by Microsoft Windows Security

**Comment by YTEC-info** [comment]
Can you list the files that Windows Defender flagged?

**Comment by sylvestersl** [comment]
Yes, 2 virus

PWS:Win32/Ymacco.AA42
Trojan:AndroidOS/Multiverze

Thanks

Obtenir Outlook pour Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by sylvestersl** [comment]
Sorry, all exe files.

Obtenir Outlook pour Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by YTEC-info** [comment]
It seems that Windows Defender is shooting in all directions with anything that is not from the store lol

The problem is that all Windows software for the CH341A is not open source, so it is not possible to audit anything to check if there is anything malicious. And trusting what antivirus x or y says is not the best option, even if it says that all .exe files are safe does not mean that they really are.

In my own experience, I used all the software and did not get any malicious or strange behavior.

You are not the first person to open an issue claiming this. To solve this issue I am considering moving all Windows software and drivers to a separate branch from the main one, with the warning that they are closed source and unaudited binaries, which may or may not have security risks because of this. Anyone who clones the repository will be warned in the README.

I will keep this issue open to make it more visible.

**Comment by shikulja** [comment]
![image](https://github.com/user-attachments/assets/82caf1ca-322a-4fca-a5e3-2abff8c7fbbb)
![image](https://github.com/user-attachments/assets/4f5a7e82-a37c-4c28-8d3d-92179136d914)

It looks like the uploaded files were infected. This is the first time I’ve come across different programs written by different people for which anti-virus protection responds in the same way.

**Comment by lvent** [comment]
Malwarebytes reports them, virustotal reports them, kaspersky reports them, yet its defenders fault? These files are infected, period. https://github.com/YTEC-info/CH341A-Softwares/issues/7

**Comment by Feay** [comment]
not true, most of them listed are in fact open source, or derivatarives of open source.  "all Windows software for the CH341A is not open source" 
More over ignore [sylvestersl](https://github.com/sylvestersl) is a fool.

**Comment by Feay** [comment]
Look at the development history, only neoprogrammer was developed by a Russian, but was based on ASprogrammer which is open source on the manufacture's own web site, IF you can be bothered to read Chinese

**Comment by smartonaut** [comment]
Virus: Trojan.Generic.6753864 (Engine A)

Es wurde versucht, auf eine infizierte
Datei zuzugreifen.

Datei: CH341SER.EXE
Verzeichnis: D:\Install\Programierung\CH341A-Softwares-main\Drivers\WINDOWS

![Screenshot 2024-12-09 183637](https://github.com/user-attachments/assets/3de724cb-fbe1-4ef6-8fc6-29f083c3cfff)

**Comment by YTEC-info** [comment]
To centralize the discussion on this, participate in issue #18
